source/consts/application.ts

```diff
@@ -0,0 +1,6 @@
+declare const __VERSION__: string;
+
+export const APP_NAME = 'backtrace-node';
+export const VERSION = '1.2.1';
```

Comment:
did we read the version from the package.json file in the past?

Reply:
In the backtrace-js package that was done with a webpack plugin, but this package doesn't have that (..yet ?)

Reply:
Yes we don't have it and this is why we read it from the package.json.

Reply:
So what should I do? Is current approach okay? Or should I add code to read it in from the package.json?